scripts/test-runner.js

```diff
@@ -480,6 +480,9 @@ function runTest(name, code, options: PrepackOptions, args) {
     if (compileJSXWithBabel) {
       expectedCode = transformWithBabel(expectedCode, ["@babel/plugin-transform-react-jsx"]);
     }
+    // For some tests (e.g. nested optimized function definition) there is no way to pass lint due to undefined global
```

Comment:
Can we teach the linter that __optimize is okay? @trueadm ?

Reply:
We can probably add it to the whitelist: https://github.com/facebook/prepack/blob/master/scripts/lint-config.js#L14